models/Element/AbstractElement.php

```diff
@@ -222,20 +224,27 @@ public function isAllowed($type, ?Model\User\AbstractUser $user = null)
         }
 
         //everything is allowed for admin
+        /** @var Model\User $user */
         if ($user->isAdmin()) {
```

Comment:
I think we should change the method param. If someone put a role it would be occurred an error. The Dao method also accept only User objects